plugins/woocommerce/src/Admin/API/Reports/Customers/Stats/Controller.php

```diff
@@ -83,7 +85,7 @@ protected function prepare_reports_query( $request ) {
 	 */
 	public function get_items( $request ) {
 		$query_args      = $this->prepare_reports_query( $request );
-		$customers_query = new Query( $query_args );
+		$customers_query = new Query( $query_args, 'customers-stats' );
```

Comment:
We [do use here `CustomersQuery`](https://github.com/woocommerce/woocommerce/blob/add/generic-queue/plugins/woocommerce/src/Admin/API/Reports/Customers/Stats/Controller.php#L10:L10) (`Customers/Query.php`, no `/Stats`) that extends `GenericQuery`.
So we could use the same default query vars as Customers but set a different custom name for the data store and filters.

Reply:
Ah I see. Didn't realise the `Customers/Stats/Controller` is using the query from `Customers/Query` instead of `Customers/Stats/Query`. Now I understand it's because the default query vars for `Customers/Query` and `Customers/Stats/Query` are identical so we can reuse the same Query class. Unlike `Orders/Query` and `Orders/Stats/Query`, their default query vars are different so both of them extends the GenericQuery and override the `get_default_query_vars()` method.